utility/block.go

```diff
@@ -103,7 +103,7 @@ func (u *UtilityContext) GetAppHash() ([]byte, typesUtil.Error) {
 
 // HandleByzantineValidators handles the validators who either didn't sign at all or disagreed with the 2/3+ majority
 func (u *UtilityContext) HandleByzantineValidators(lastBlockByzantineValidators [][]byte) typesUtil.Error {
-	latestBlockHeight, err := u.GetLatestHeight()
+	latestBlockHeight, err := u.GetHeight()
```

Comment:
Should we also rename the variable to `height` or `blockHeight` given the function definition change?

Ditto other occurences.